www/js/overlay.js

```diff
@@ -176,6 +176,48 @@ const resizeButtonFactory = (increaseFunc, decreaseFunc) =>
     ),
   );
 
+const closeAllDropDowns = () => {
+  document
+    .querySelectorAll('.options-container.visible')
+    .forEach(el => el.classList.remove('visible'));
+};
+
+const toggleDropDown = $select => {
+  closeAllDropDowns();
+  const $options = $select.querySelector('.options-container');
+  $options.classList.add('visible');
+  $options.style.top = `${
+    window.innerHeight < $options.offsetHeight + $select.offsetTop
+      ? -$options.offsetHeight
+      : $select.querySelector('.select-value').offsetHeight
+  }px`;
+};
+
+window.addEventListener('click', e => {
+  if (!e.target.classList.contains('select-value')) {
+    closeAllDropDowns();
+  } else {
+    toggleDropDown(e.target.parentElement);
+  }
+});
+
+const fontSwitch = (e, font, propName) => {
+  e.target.parentElement.parentElement.querySelector('.select-value').innerHTML = font;
+  overlayVars[propName] = font;
+  savePrefs();
+};
+
+const fontListFactory = (list, propName) => {
+  const options = list.map(font =>
+    h(`div.option`, { onclick: e => fontSwitch(e, font, propName) }, font),
+  );
+  return h(
+    'div.custom-select',
```

Comment:
could have probably extracted this component to separate file. can follow up on this later.